src/core/symbology-ng/qgssymbollayerutils.cpp

```diff
@@ -2514,7 +2515,15 @@ bool QgsSymbolLayerUtils::functionFromSldElement( QDomElement &element, QString
     return false;
   }
 
+  // check if it is a single string value
+  if ( elem.hasChildNodes() &&
+       elem.firstChild().nodeType() == QDomNode::TextNode )
+  {
+    function = elem.firstChild().nodeValue();
```

Comment:
right... is simple and possibile... I'll do it! tnx @m-kuhn to point it out.
In this moment assume that the expression build does not consider a literal constant othan expressed with ogc:Literal